.yarnrc.yml

```diff
@@ -45,4 +42,4 @@ plugins:
     spec: "@yarnpkg/plugin-interactive-tools"
   - ./.hooks/plugin-warning-logger.js
 
-yarnPath: .yarn/releases/yarn-3.2.0.cjs
+yarnPath: .yarn/releases/yarn-3.2.4.cjs
```

Comment:
This was not strictly necessary, and is just because the command I was given happens to default to the latest applicable version. It seems to work, so I also was inclined to keep it in the spirit of keeping things up to date.